packages/generator/src/sdk-metadata/sdk-metadata-types.ts

```diff
@@ -85,3 +93,34 @@ export type DateTimeString = string;
  * String fields containing a text formatted with multiple lines.
  */
 export type MultiLineText = string;
+
+export interface ServiceStatus {
+  /**
+   * certified -> Published lib, verified -> Generation worked
+   */
+  status: 'certified' | 'verified' | 'unknown';
+  /**
+   * Detailed text what the serviceStatus means.
+   * @memberof Client
+   */
+  statusText: string;
+  /**
+   * Getting started text, depends on the service status
+   */
+  gettingStartedText: string;
+}
+
+interface LinkWithName {
+  url: UrlString;
+  name: string;
+}
+
+export interface EmergencyObject {
+  status: string;
+  description: string;
+}
+
+export interface InstructionWithText {
+  instructions: MultiLineText;
+  text: string;
+}
```

Comment:
I think those could benefit from API docs. I don't fully get what InstructionsWithText (so what is `text`? Maybe `title`?).